yb_migrate/src/migration/postgres.go

```diff
@@ -53,10 +53,9 @@ func PgDumpExtractSchema(source *utils.Source, exportDir string) {
 			source.Port, source.DBName, SSLQueryString, exportDir)
 	}
 
+	log.Infof("Running command:%s\n", prepareYsqldumpCommandString)
 	preparedYsqldumpCommand := exec.Command("/bin/bash", "-c", prepareYsqldumpCommandString)
```

Comment:
Not meant for this PR, but consider renaming "prepareYsqldumpCommandString" -> "preparePgDumpCommandString" for the sake of correctness in variable naming.